### PR TITLE
[14.0] [REF] Refactor account_reconciliation_widget files

### PR DESCRIPTION
**Pull request by PieterPaulussen**
Update the module files and filenames to align with more recent OCA guidelines

### CODE REVIEW

File: account_reconciliation_widget/__manifest__.py [hunk 1]
```diff
@@ -13,9 +13,9 @@
     "data": [
         "security/ir.model.access.csv",
         "views/assets.xml",
-        "views/account_view.xml",
-        "views/account_bank_statement_view.xml",
-        "views/account_journal_dashboard_view.xml",
+        "views/account_bank_statement.xml",
```
Comment (pedrobaeza) [inline account_reconciliation_widget/__manifest__.py:16]: Guidelines indicate to put the `_view` or `_views` suffix.

File: account_reconciliation_widget/models/__init__.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
-from . import account_move
+from . import account_move_line
 from . import account_bank_statement
+from . import account_bank_statement_line
```
Comment (pedrobaeza) [inline account_reconciliation_widget/models/__init__.py:3]: You shouldn't split the children object in one file.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
OCA guidelines don't recommend the changes you are proposing:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst

**Comment by PieterPaulussen** [comment]
@pedrobaeza You are correct. My apologies!

**Pull request closed without merging**